README.rst

```diff
@@ -2,17 +2,19 @@
 os-net-config
 =============
 
+A declarative network configuration tool for hosts.
+
 Team and repository tags
 ------------------------
 
-.. image:: https://governance.openstack.org/tc/badges/os-net-config.svg
-    :target: https://governance.openstack.org/tc/reference/tags/index.html
+.. image:: https://governance.openstack.org/tc/badges/openapi.svg
+    :target: https://governance.openstack.org/tc/reference/projects/
 
 Overview
 --------
 
 ``os-net-config`` is a host network configuration tool which supports multiple
-backend configuration providers.
+backend configuration providers (One of: ifcfg, eni, nmstate, iproute).
 
 * Documentation: https://docs.openstack.org/os-net-config/latest
```

Comment:
It's not important for this PR, but we should probably move the latest docs to GitHub and point there in a follow-up